test/form/samples/supports-es5-shim/_expected.js

```diff
@@ -180,7 +180,7 @@ function requireEs5Shim () {
 		            var n = +num;
 		            if (isActualNaN(n)) {
 		                n = 0;
-		            } else if (n !== 0 && n !== (1 / 0) && n !== -(1 / 0)) {
+		            } else if (n !== 0 && n !== (1 / 0) && n !== -Infinity) {
```

Comment:
`1/0` is actually shorter than `Infinity`. But on the other hand, a minifier might also do this transformation.

Reply:
Yes, I'm also hesitating about whether to keep the `-(1/0)`. If we do this, we'll have to convert all the `-Infinity` to `-(1/0)` as well.

Reply:
Personally, I’d prefer not to convert `-Infinity`. Terser does that, but esbuild doesn’t yet.